apm/profiling/intro-profiling.rst

```diff
@@ -12,7 +12,6 @@ Introduction to AlwaysOn Profiling for Splunk APM
 
    profiling-use-case-library
    get-data-in-profiling
-   spans-stack-traces
```

Comment:
See my previous comment. Have we removed a feature or was this added by mistake in the previous version of the documentation?

Reply:
> The spans-stack-traces.srt page and the respective references have been removed as the feature has been deprecated.